### PR TITLE
feat: add rating component

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added Rating Component**
> Usage

rating

### CODE REVIEW

File: components/rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+.rating:not(:checked) > label:hover ~ label {
+    color: #eab308;
+}
+
+.rating > input:checked + label:hover,
+.rating > input:checked ~ label:hover,
+.rating > label:hover ~ input:checked ~ label,
+.rating > input:checked ~ label:hover ~ label {
+    color: #eab308;
+}
```
Comment (Richa-rani02) [inline components/rating/rating.css]: Suggestion : better to define color variable

**Review by Richa-rani02 — CHANGES_REQUESTED** [review]

File: components/rating/rating.css (hunk 1 above)
Comment (nerdyspook) [reply components/rating/rating.css]: Thanks I almost forgot to add a variable for this one.

**Review by Richa-rani02 — APPROVED** [review]

**Comment by Richa-rani02** [comment]
done

**Review by KanekarSnehal — APPROVED** [review]
Code looks good, coding practices followed.

**Pull request merged**